requirements.txt

```diff
@@ -1,2 +1,3 @@
 requests==2.11.1
 monotonic==1.2
+git+https://github.com/uktrade/directory-signature-auth.git@ED-201-sigauth-library#egg=sigauth
```

Comment:
to change `ED-201-sigauth-library` to `0.1.0` when the signature library is ready